app/components/using_these_materials_component.rb

```diff
@@ -14,6 +14,15 @@ def repository_url
   end
 
   def restrictions
-    document.collection.fetch('accessrestrict_html_tesm', ['']).first.html_safe # rubocop:disable Rails/OutputSafety
+    raw_html = document.collection.fetch('accessrestrict_html_tesm', ['']).join
```

Comment:
I think we want to join with a space so separate fields don't get jammed together.
```suggestion
    raw_html = document.collection.fetch('accessrestrict_html_tesm', ['']).join(' ')
```
<img width="379" alt="Screenshot 2025-05-22 at 8 52 17 AM" src="https://github.com/user-attachments/assets/ffc7b7d2-7eee-438f-a40e-271d7b03a669" />